src/framework/preferences/js/PrefsEditor.js

```diff
@@ -435,7 +446,9 @@ var fluid_2_0 = fluid_2_0 || {};
         var bindHandlers = function (that) {
             var saveButton = that.locate("save");
             if (saveButton.length > 0) {
-                saveButton.click(that.saveAndApply);
+                saveButton.click(function () {
+                    that.saveAndApply("");
```

Comment:
Should we leave the default as it has been, that is, only saving the preferences and not the entire model?